# Generated by Django 3.0.3 on 2023-11-24 08:53

import datetime
from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Fetcher',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=255)),
                ('enable', models.BooleanField(default=True)),
                ('sum_proxies_cnt', models.IntegerField(default=0)),
                ('last_proxies_cnt', models.IntegerField(default=0)),
                ('last_fetch_date', models.TextField(blank=True, null=True)),
            ],
            options={
                'db_table': 'fetchers',
                'managed': True,
            },
        ),
        migrations.CreateModel(
            name='Proxy',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('fetcher_name', models.CharField(max_length=255)),
                ('protocol', models.CharField(max_length=255)),
                ('ip', models.CharField(max_length=255)),
                ('port', models.IntegerField()),
                ('validated', models.BooleanField(default=False)),
                ('latency', models.IntegerField(blank=True, null=True)),
                ('validate_date', models.TextField(blank=True, null=True)),
                ('to_validate_date', models.TextField(default=datetime.datetime(2023, 11, 24, 16, 53, 39, 511519))),
                ('validate_failed_cnt', models.IntegerField(default=0)),
            ],
            options={
                'db_table': 'proxies',
                'managed': True,
            },
        ),
    ]
